example/src/new_api/swipeable/AppleStyleSwipeableRow.tsx

```diff
@@ -135,16 +137,16 @@ export default function AppleStyleSwipeableRow({
       enableTrackpadTwoFingerGesture
       leftThreshold={30}
       rightThreshold={40}
-      renderLeftActions={(_, progress) =>
+      renderLeftActions={(_: any, progress: SharedValue<number>) =>
         renderLeftActions(_, progress, swipeableRow)
       }
-      renderRightActions={(_, progress) =>
+      renderRightActions={(_: any, progress: SharedValue<number>) =>
         renderRightActions(_, progress, swipeableRow)
       }
-      onSwipeableWillOpen={(direction) => {
+      onSwipeableWillOpen={(direction: any) => {
         console.log(`Opening swipeable from the ${direction}`);
       }}
-      onSwipeableWillClose={(direction) => {
+      onSwipeableWillClose={(direction: any) => {
```

Comment:
Why set the types explicitly here, shouldn't they be inferred?

And if they are necessary here, why `any`?

Reply:
these new eslint rules flag inferring as a bad practice.
`any` is used in cases we don't care about the type of said variable, in our case it's only used when the variable either isn't used, or is printed